packages/windows_open_package/correlation_rules/mitre_attck_priv_esc/Start_mmc_exe_from_suspicious_binary_path/metainfo.yaml

```
@@ -0,0 +1,28 @@
+ContentAutoName: Start_mmc_exe_from_suspicious_binary_path
+ExpertContext:
+    Created: 02.07.2024
+    Updated: 02.07.2024
+    KnowledgeHolders:
+        - driverenok
+    Usecases:
+        - Ататкующие могут изменять переменную %SYSTEMROOT% чтобы обходить UAC и запускать фейковый процесс mmc.exe
+    Falsepositives: []
+    References:
+        - https://bytecode77.com/performance-monitor-privilege-escalation
+    Improvements: []
+    DataSources:
+        - Provider: Microsoft-Windows-Security-Auditing
+          EventID:
+              - 4688
+        - Provider: Microsoft-Windows-Sysmon
+          EventID:
+              - 1
+ObjectId: SEC-CR-61536543
+ContentRelations:
+    Implements:
+        ATTACK:
+            privilege-escalation:
+                - T1548.002
```

Запуск фейкового mmc.exe больше похоже на _Masquerading: Match Legitimate Name or Location_. Просто в связке с подменой переменной окружения) Но в целом могут и без подмены переменной окружения мимикрировать под легитимный процесс :)

```suggestion
            defense-evasion:
                - T1036.005
```